CHANGES.rst

```diff
@@ -1,5 +1,15 @@
 .. currentmodule:: flask
 
+Version 2.1.0
+-------------
+
+Unreleased
+
+-   Update Click dependency to >= 8.0.
+-   Fixed the issue where typing requires template global
+    decorators to accept functions with no arguments. :issue:`4098`
```

Comment:
This should be part of the 2.0.2 changelog

Reply:
sure